build/repo.targets

```diff
@@ -55,4 +56,20 @@
     <RemoveDir Directories="$(ProductPackagesSource)" />
   </Target>
 
+  <Target Name="PublishNpmModules">
+    <PropertyGroup>
+      <AuthTokenSetting>$(NpmRegistry.Replace("https:", "")):_authToken</AuthTokenSetting>
+    </PropertyGroup>
+    
+    <ItemGroup>
+      <NpmModule Include="$(ArtifactsDir)build\*.tgz" />
+    </ItemGroup>
+    
+    <Exec Command="npm config set $(AuthTokenSetting) $(APIKEY)" />
+    <Exec Command="npm publish --registry $(NpmRegistry) %(NpmModule.Identity)" ContinueOnError="true">
```

Comment:
You might want to use `IgnoreStandardErrorWarningFormat` if the output is causing the build to fail. It'll use non-zero exit codes to fail so you can remove the `ExitCodes` check. Unless the intent is to publish as many packages as we want (which seems unusual)